agent_api_rest/src/lib.rs

```diff
@@ -63,9 +68,14 @@ pub fn app(state: ApplicationState) -> Router {
         .route(&path("/auth/token"), post(token))
         .route(&path("/openid4vci/credential"), post(credential))
         // Agent Verification Preparations
-        .route(&path("/v1/authorization_requests"), post(authorization_requests))
         .route(
-            &path("/v1/authorization_requests/:authorization_request_id"),
+            &path(&format!("/{API_VERSION}/authorization_requests")),
```

Comment:
Maybe you make a const fn that will have 1 argument path_suffix and will return the complete url? will make it a bit more compact.

Reply:
hmm the problem here though is that not all endpoints have  the API version prefix :thinking:

Reply:
but I can solve it by using this probably: https://docs.rs/axum/latest/axum/struct.Router.html#method.nest

I'll give that a go :+1:

Reply:
Yes you can use that or maybe have a `path_ver` parallel to it.

Reply:
Ok I've changed it to this now:
```rust
    pub const API_VERSION: &str = "/v0";

    ...

    Router::new()
        .nest(
            &path(API_VERSION),
            Router::new()
                // Agent Issuance Preparations
                .route(
                    "/configurations/credential_configurations",
                    post(configurations::credential_configurations),
                )
                .route("/credentials", post(credentials))
                .route("/credentials/:credential_id", get(get_credentials))
                .route("/offers", post(offers))
                // Agent Verification Preparations
                .route("/authorization_requests", post(authorization_requests))
                .route(
                    "/authorization_requests/:authorization_request_id",
                    get(get_authorization_requests),
                ),
        )
```

which looks cleaner I think